UnityProject/Assets/Scripts/Objects/Doors/Modules/EmagInteractionModule.cs

```diff
@@ -18,8 +18,13 @@ protected override void Awake()
 
 		public override void ClosedInteraction(HandApply interaction, HashSet<DoorProcessingStates> States)
 		{
-			var ItemStorage = interaction.Performer.GetComponent<DynamicItemStorage>();
-			EmagChecks(ItemStorage, interaction, States);
+			if (interaction != null)
```

Comment:
wait, are you sure this works to prevent emag interaction from running? If you use a crowbar, interaction will still exist on won't be null.